scripts/apps/index.js

```diff
@@ -22,6 +22,7 @@ import 'apps/monitoring';
 import 'apps/settings';
 import 'apps/ingest';
 import 'apps/search-providers';
+import 'apps/web-publisher';
```

Comment:
I think it might be best to remove the `if` condition in `scripts/apps/web-publisher/index.js` line 22 (about the config), and instead do it here, like:
```js
/* globals __SUPERDESK_CONFIG__: true */
const appConfig = __SUPERDESK_CONFIG__;
const withPublisher = typeof appConfig.publisher !== 'undefined';

if (withPublisher) {
    require('apps/web-publisher');
}

if (appConfig.features.useTansaProofing) {
    require('apps/tansa');
}
```

And do you mind removing the tansa logic from [scripts/index.js#L16-L18](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/index.js#L16-L18) ? It makes more sense for it to be here.